optimum/exporters/onnx/config.py

```diff
@@ -150,9 +173,47 @@ class AudioOnnxConfig(OnnxConfig):
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyAudioInputGenerator,)
 
 
-class TextAndAudioOnnxConfig(OnnxSeq2SeqConfigWithPast):
+class AudioToTextOnnxConfig(OnnxSeq2SeqConfigWithPast):
```

Comment:
Could we align the names of the config with the `AutoModels`.

For text to text. we have  - `TextSeq2Seq...`
For speech to text, -> `SpeechSeq2Seq`
For Vision to text -> `Vision2Seq`

Reply:
For me it is more like `TextAndVision` is because it does not care about the direction, but for instance `AudioToText` means that the encoder side will be audio, and the decoder side will be text.

I think here there is the modality, and the seq2seq part, for instance, we could have:

- `SpeechSeq2Seq` meaning "Speech to speech"
- `Speech2Text` meaning "Speech to text" implying that it's seq2seq

I guess, we can discuss about it on another PR, but I agree that specifying a standard would be good!

Reply:
Created the issue => #597

Reply:
great